components/nexus-repository/src/main/java/org/sonatype/nexus/repository/view/ContentHeadersHandler.java

```diff
@@ -10,18 +10,12 @@
  * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
  * Eclipse Foundation. All other trademarks are the property of their respective owners.
  */
-package org.sonatype.nexus.repository.maven.internal;
+package org.sonatype.nexus.repository.view;
```

Comment:
There is a view.handlers package where standard handles are meant to live, why aren't you using that?